### PR TITLE
Fix-For-ADFS-and-Windows-Event-Logs

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created in order of updating modeling rules and parsing rules that will target ADFS and Windows Event Logs to distinct datasets.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 8.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/MicrosoftADFS/README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 This pack includes XSIAM content.
 
 ​
-Note: The logs will be stored in the dataset named *microsoft_windows_raw*.
+**Notes:** 
+1. The logs will be stored in the dataset named *microsoft_adfs_raw*.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/README.md:7]:
```suggestion
- The logs will be stored in the dataset called *microsoft_adfs_raw*.
```

File: Packs/MicrosoftADFS/README.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
+2. The default content provided in this pack (including Winlogbeat templates) should be used as is.  We cannot support any changes made to this content.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/README.md:8]:
```suggestion
- The default content provided in this pack (including Winlogbeat templates) should be used as is.  We cannot support any changes made to this content.
```

File: Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+#### Parsing Rules
+
+##### Microsoft ADFS Collection Parsing Rule
+
+- Updated the Parsing Rule to point to a new dataset "microsoft_adfs_raw".
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md:6]:
```suggestion
Updated the parsing rule to point to a new "microsoft_adfs_raw" dataset.
```

File: Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+
+#### Modeling Rules
+
+##### Microsoft ADFS Collection
+
+- Updated the Modeling Rule with the correct dataset name.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftADFS/ReleaseNotes/1_0_14.md:13]:
```suggestion
Updated the modeling rule with the correct dataset name.
```

File: Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_6.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Microsoft Windows Events Parsing Rule
+
+- Updated the filter for the Parsing Rule to exclude "AD FS" logs.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_6.md:6]:
```suggestion
Updated the filter for the parsing rule to exclude "AD FS" logs.
```

File: Packs/MicrosoftADFS/README.md [hunk 6]
```diff
@@ -3,11 +3,11 @@
 This pack includes XSIAM content.
```
Comment (evisochek) [inline Packs/MicrosoftADFS/README.md:3]: @nkanon please add instructions for winlogbeat as well.

**Review by evisochek — APPROVED** [review]

File: Packs/MicrosoftADFS/README.md (hunk 6 above)
Comment (nkanon) [reply Packs/MicrosoftADFS/README.md:3]: Added. Thanks Boss

**Pull request merged**